### PR TITLE
Adding Scikit and Tensorflow serving example into local ci

### PR DESCRIPTION
**Pull request by anjalirai-intel**
ubuntu 20.04: http://ba5sapp0350:8080/view/Old%20Graphene%20Jobs/job/anjali_gsgx_ubuntu/290/console
centos 8: http://ba5sapp0350:8080/view/Old%20Graphene%20Jobs/job/anjali_gsgx_ubuntu/282/console
ubuntu 22.04: http://ba5sapp0350:8080/view/Old%20Graphene%20Jobs/job/anjali_gsgx_ubuntu/288/consoleFull
ubuntu 18.04: http://ba5sapp0350:8080/view/Old%20Graphene%20Jobs/job/anjali_gsgx_ubuntu/285/consoleFull

### CODE REVIEW

File: ci/dockerfile/ubuntu20.04.dockerfile [hunk 1]
```diff
@@ -1,5 +1,13 @@
 FROM ubuntu:20.04
 
+RUN apt update && DEBIAN_FRONTEND=noninteractive apt install -y curl gnupg2 binutils
```
Comment (jinengandhi-intel) [inline ci/dockerfile/ubuntu20.04.dockerfile:3]: Same as above

File: ci/dockerfile/ubuntu22.04.dockerfile [hunk 2]
```diff
@@ -1,5 +1,13 @@
 FROM ubuntu:22.04
 
+RUN apt update && DEBIAN_FRONTEND=noninteractive apt install -y curl gnupg2 binutils
```
Comment (jinengandhi-intel) [inline ci/dockerfile/ubuntu22.04.dockerfile:3]: Same as above.

File: ci/dockerfile/ubuntu18.04.dockerfile [hunk 3]
```diff
@@ -1,6 +1,14 @@
 # Start with 18.04
 FROM ubuntu:18.04
 
+RUN apt update && DEBIAN_FRONTEND=noninteractive apt install -y curl gnupg2 binutils
```
Comment (jinengandhi-intel) [inline ci/dockerfile/ubuntu18.04.dockerfile:4]: Can we put this workload specific installation in Makefile?

Comment (anjalirai-intel) [reply ci/dockerfile/ubuntu18.04.dockerfile:4]: I will make the change

File: ci/dockerfile/ubuntu22.04.dockerfile [hunk 4]
```diff
@@ -97,14 +98,17 @@ RUN git clone https://github.com/giltene/wrk2.git \
 # NOTE about meson version: we support "0.56 or newer", so in CI we pin to latest patch version of
 # the earliest supported minor version (pip implicitly installs latest version satisfying the
 # specification)
+RUN python3 -m pip install --upgrade pip --user
```
Comment (jinengandhi-intel) [inline ci/dockerfile/ubuntu22.04.dockerfile:101]: Same as above.

File: ci/dockerfile/ubuntu20.04.dockerfile [hunk 5]
```diff
@@ -101,13 +99,19 @@ RUN git clone https://github.com/giltene/wrk2.git \
 # NOTE about meson version: we support "0.56 or newer", so in CI we pin to latest patch version of
 # the earliest supported minor version (pip implicitly installs latest version satisfying the
 # specification)
+RUN python3 -m pip install --upgrade pip --user
```
Comment (jinengandhi-intel) [inline ci/dockerfile/ubuntu20.04.dockerfile:102]: Do we need this here? I thought this was only for Ubuntu18.04?

File: ci/dockerfile/ubuntu22.04.dockerfile (hunk 4 above)
Comment (anjalirai-intel) [reply ci/dockerfile/ubuntu22.04.dockerfile:101]: We do need it for installing scikit packages. Without pip upgrade, it does not found the package

**Review by jinengandhi-intel — APPROVED** [review]

**Pull request merged**